projects/1.16/assets/entangled/entangled/lang/zh_cn.json

```
@@ -1,23 +1,24 @@
 {
-  "block.entangled.block": "纠缠方块",
-  "item.entangled.item": "纠缠绑定器",
-  "entangled.entangled_block.info.ranged_same_dimension": "可以绑定到同一维度内最远%d格的方块",
-  "entangled.entangled_block.info.infinite_same_dimension": "可以绑定到同一维度内的方块",
-  "entangled.entangled_block.info.ranged_other_dimension": "可以绑定到同一维度内最远%d格的方块或其他维度的方块",
-  "entangled.entangled_block.info.infinite_other_dimension": "可以绑定到其他方块",
   "entangled.entangled_block.info.bound": "已绑定至位于%5$s维度（%2$d，%3$d，%4$d）的%1$s",
-  "entangled.entangled_block.unbind": "方块已解绑！",
-  "entangled.entangled_block.no_selection": "未选择方块！",
   "entangled.entangled_block.self": "不能绑定方块本身！",
-  "entangled.entangled_block.bind": "方块已绑定！",
-  "entangled.entangled_block.wrong_dimension": "目标方块必须在同一维度！",
+  "config.jade.plugin_entangled.entangled_block_component": "纠缠方块",
+  "entangled.entangled_block.info.ranged_same_dimension": "可以绑定到同一维度内最远%d格的方块",
   "entangled.entangled_block.too_far": "目标方块距离过远！",
+  "entangled.entangled_binder.select": "方块已选择！",
+  "entangled.entangled_block.info.infinite_other_dimension": "可以绑定到其他方块",
+  "entangled.block.block": "纠缠方块",
+  "entangled.item.item": "纠缠绑定器",
+  "entangled.entangled_block.wrong_dimension": "目标方块必须在同一维度！",
+  "entangled.entangled_block.no_selection": "未选择方块！",
   "entangled.entangled_binder.info": "可以将纠缠方块与其他方块绑定",
-  "entangled.entangled_binder.info.target.unknown": "已连接到位于%4$s维度（%1$d，%2$d，%3$d）的方块",
   "entangled.entangled_binder.info.target.known": "已连接到位于%5$s维度（%2$d，%3$d，%4$d）的%1$s",
-  "entangled.entangled_binder.select": "方块已选择！",
-  "entangled.entangled_binder.clear": "连接已清除！",
-  "entangled.waila.bound_same_dimension": "已绑定至（%2$d，%3$d，%4$d）的%1$s",
+  "entangled.entangled_block.bind": "方块已绑定！",
+  "entangled.entangled_block.info.infinite_same_dimension": "可以绑定到同一维度内的方块",
+  "entangled.entangled_block.info.ranged_other_dimension": "可以绑定到同一维度内最远%d格的方块或其他维度的方块",
+  "entangled.entangled_block.unbind": "方块已解绑！",
   "entangled.waila.bound_other_dimension": "已绑定至位于%5$s维度（%2$d，%3$d，%4$d）的%1$s",
```

我认为是这样的，
```suggestion
  "entangled.waila.bound_same_dimension": "已绑定到位于%5$s的%1$s（%2$d，%3$d，%4$d）",
```

坐标应该指示的是绑定目标的位置，维度本身怎么会有坐标
`维度`加不加应该无所谓

---

已改为该格式